### ISSUE TITLE
not good in voc dataset

### ISSUE THREAD

**Issue by pengfeidip** [yhenon/pytorch-retinanet#64]
- train: voc2007 trainval + voc2012 trainval
- test: voc 2007 test
- mAP in train 0.75
- mAP in test 0.58 

then performance is not changed， why？

**Comment by NineBall9** [comment]
how many epoches have you trained?

**Comment by pengfeidip** [comment]
@NineBall9  60 epochs


![image](https://user-images.githubusercontent.com/23066797/53846910-b9e17080-3fe9-11e9-85a7-60d00558292f.png)
>the left image is loss curve by epoch; the right image is testing mAP by epoch

**Comment by pengfeidip** [comment]
@NineBall9  and I think the problem is the params of focal loss, but I don't have enough time to tune the params

**Comment by NineBall9** [comment]
Maybe you are right. I got a similar result.
